griptape/drivers/vector/base_vector_driver.py

```diff
@@ -1,7 +1,7 @@
 from abc import ABC, abstractmethod
 from concurrent import futures
 from dataclasses import dataclass
-from typing import Optional
+from typing import Optional, List, Tuple, Dict
```

Comment:
Since we support Python 3.9+ only, there is no need for importing types. Per [PEP 585](https://peps.python.org/pep-0585/), we can use built-in types directly, which we do everywhere else in the framework.